admin/acceptance.py

```diff
@@ -449,11 +449,14 @@ def main(reactor, args, base_path, top_level):
             make_dispatcher(reactor),
             configure_cluster(control_node=nodes[0], agent_nodes=nodes,
                               certificates=certificates))
+
+        control_node = nodes[0]
```

Comment:
Ah.  `nodes[0]` is passed to `configure_cluster` two lines above.  Please move this assignment up and use `control_nodes` as the value to pass to `configure_cluster` to make it more clear we're setting it up this way.